java/tests/name/fraser/neil/plaintext/Speedtest.java

```diff
@@ -13,15 +13,13 @@
 
 package name.fraser.neil.plaintext;
 
-import name.fraser.neil.plaintext.diff_match_patch;
-
 import java.io.BufferedReader;
 import java.io.FileReader;
 import java.io.IOException;
 
 public class Speedtest {
 
-  public static void main(String args[]) {
+  public static void main(String args[]) throws IOException {
```

Comment:
It would be better to use Java Microbenchmark Harness (JMH) instead. It is used for building, running, and analyzing nano/micro/milli/macro benchmarks and produces reliable results.